include/veriblock/storage/block_repository_rocks.hpp

```diff
@@ -232,6 +231,63 @@ class BlockRepositoryRocks : public BlockRepository<Block> {
     }
     return result;
   }
+
+ private:
+  class CursorRocks : public Cursor<height_t, stored_block_t> {
+   private:
+    std::vector<rocksdb::Iterator*> rocks_its;
+
+   public:
+    CursorRocks(const BlockRepositoryRocks<stored_block_t>& repo) {
+      repo._db->NewIterators(
```

Comment:
I think it may fail here. Do something about it.

Reply:
Generally it can not fail here, even if we will have an empty database everything will be ok